File: python/lista-3/ex4.py
ano = int(input("Digite o ano: "))

if ano % 400 == 0:
    print("Parabens é bissexto")

elif (ano % 100 == 0):

    print("Nao é bissexo")

elif (ano % 4 == 0):

    print("Parabens é bissexto")

else: print("Não é bissexo")

File: cpp/lista-3/ex4.cpp
#include <iostream>

int main() {

    int a, b, c;

    std::cout<<"Digite o primeiro lado: ";
    std::cin>>a;

    std::cout<<"Digite o segundo lado: ";
    std::cin>>b;

    std::cout<<"Digite o terceiro lado: ";
    std::cin>>c;

    if ((a+b < c) or (b+c < a) or (a+c < b)) {
        std::cout<<"Nao e um triangulo"<<std::endl;

    } else if (a == b and a == c) {
        std::cout<<"Parabens, e equilatero"<<std::endl;

    } else if (a == b or a == c or b == c) {
        std::cout<<"Parabens, e isosceles"<<std::endl;

    } else { 
        std::cout<<"Parabens, e escaleno"<<std::endl;

    }

    return 0;
}